even_odd_lists.cpp

```diff
@@ -0,0 +1,38 @@
+/* This C++ program gets a length and a list of numbers
+from the user and prints the odd and even numbers separately.
+*/
+
+#include <iostream>
+
+using namespace std;
+
+int main() {
+    int length;
+    cout << "How many numbers? ";
+    cin >> length;
+
+    int numbers[length];
+
+    for (int i = 0; i < length; ++i) {
+        cout << "Number " << i + 1 << ": ";
+        cin >> numbers[i];
+    }
+
+    cout << "\nEven numbers: ";
+    for (int i = 0; i < length; ++i) {

```

Comment:
You're using {} for a loop which contains only a single command. This is not necessary. see example_1.cpp of this assignment.